ISSUES.md

```diff
@@ -0,0 +1,225 @@
+    
+    Use bullets and items if you have lists of items and the outcome improves overall readability.
+    
+    Use backticks to refer to class and function names, e.g. `BartModel` and `generate` as these stand out and improve the speed of a reader's comprehension.
+    
+    Try not use italics and bold text too much as these often make the text more difficult to read.
+    
+
+12. If you are cross-referencing a specific comment in a given thread or another Issue, always link to that specific comment, rather than using the Issue link. If you do the latter it could be quite impossible to find which specific comment you're referring to.
+
+    To get the link to the specific comment do not copy the url from the location bar of your browser, but instead, click the `...` icon in the upper right corner of the comment and then select "Copy Link".
+    
+    For example the first link is a link to an Issue, and the second to a specific comment in the same Issue:
+    
+    1. https://github.com/huggingface/transformers/Issues/9257
+    2. https://github.com/huggingface/transformers/Issues/9257#Issuecomment-749945162
+
+    
+13. If you are replying to a last comment, it's totally fine to make your reply with just your comment in it. The readers can follow the information flow here.
+
+    But if you're replying to a comment that happened some comments back it's always a good practice to quote just the relevant lines you're replying it. The `>` is used for quoting, or you can always use the menu to do so. For example your editor box will look like:
+
+    ```
+    > How big is your gpu cluster
+    
+    Our cluster is made of 256 gpus
+    ```
+    
+    If you are addressing multiple comments, quote the relevant parts of each before your answer. Some people use the same comment to do multiple replies, others separate them into separate comments. Either way works. The latter approach helps for linking to a specific comment.
+
+In general the best way to figure out what works the best is learn from Issues posted by other people - see which Issues get great responses and which get little to no response - observe what the posters who received great responses did differently from those who did not. 
+
+Thank you for reading this somewhat lengthy document. We would like to conclude that these are not rules, but guidelines that will help maximize the chances for us to understand what you are trying to communicate, reproduce the problem and resolve it to your satisfaction and the benefit of the whole community.
+
+If after reading this document there are remaining questions on how and why or there is a need for further elucidation, please, don't hesitate to ask "How do I?" type of questions in the following sub-section of the forums: [XXX: @sgugger].
```

Comment:
When we have a finished and ready to merge version of this, I will copy-paste it on the forums and we can link the topic here (so that will be a topic instead of a subsection).